bika/lims/browser/analyses.py

```diff
@@ -459,7 +459,7 @@ def folderitem(self, obj, item, index):
         item['class']['retested'] = 'center'
         item['result_captured'] = self.ulocalized_time(
             obj.getResultCaptureDate, long_format=0)
-        item['calculation'] = obj.getCalculation and True or False
+        item['calculation'] = obj.getCalculationUID
```

Comment:
Better change this line to `obj.getCalculationUID and True or False` so that it still contains a boolean type instead of an empty string `''` or an UID `'0666cd13ce844451b24f32ec8f5cd883'`

Reply:
The value for this field is only used to check if the analysis requires calculation, so I thought there was any difference. In fact, I thought was better to set the UID, cause it would be used later with coffescript to get the calculation directly. But yes, you are right, will do the change